docs/documentation/setup/networking.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Networking"
+sidebarTitle: "Networking"
+description: "Network configuration details for Infisical Cloud"
+---
+
+## Overview
+
+When integrating your infrastructure with Infisical Cloud, you may need to configure network access controls. This page provides the IP addresses that Infisical uses to communicate with your services.
+
+## Egress IP Addresses
+
+Infisical Cloud operates from two regions: US and EU. If your infrastructure has strict network policies, you may need to allow traffic from Infisical by adding the following IP addresses to your ingress rules. These are the egress IPs Infisical uses when making outbound requests to your services.
```

Comment:
you say egress here and then ingress down there. are they the same ips?

Reply:
they are both referring to the same IP. If you're the network admin for an org, you want to get the Egress IP of Infisical and then add it as one of the allowed IPs in your network's Ingress